pkg/recipes/configloader/environment.go

```diff
@@ -101,14 +101,16 @@ func getConfiguration(environment *v20231001preview.EnvironmentResource, applica
 		return nil, ErrUnsupportedComputeKind
 	}
 
-	providers := environment.Properties.Providers
-	if providers != nil {
-		if providers.Aws != nil {
-			config.Providers.AWS.Scope = to.String(providers.Aws.Scope)
-		}
-		if providers.Azure != nil {
-			config.Providers.Azure.Scope = to.String(providers.Azure.Scope)
-		}
+	env, err := environment.ConvertTo()
```

Comment:
Is **ConvertTo** a new function? It doesn't make a lot of sense to me. We want to convert environment to what specifically? Not in the scope of this PR, but we should check this function.

Reply:
i am using the existing convertTo() function from environment_conversions